private/highscoreList.js

```diff
@@ -0,0 +1,95 @@
+const router = require('express').Router();
+const db = require('../database/connection.js').connection;
+
+router.get('/api/highscores/user/:userHighestScore', (req, res) => {
```

Comment:
It might be a good idea that during our next meeting we could create the api table again to document which endpoints are getting what data.

Reply:
yes pls my naming is terrible